.github/workflows/test.yaml

```diff
@@ -656,7 +656,7 @@ jobs:
           replaced=${string/$search/$replace}
           echo "DATABASE_URL=$replaced" >> $GITHUB_ENV
 
-      - name: test ${{ matrix.bundler }} - ${{matrix.engine}}
+      - name: test ${{ matrix.bundler }} - ${{matrix.clientEngine}}
```

Comment:
I copied this from master, looks like some changes were made. 

TODO: port these to newer script